packages/freezed_annotation/lib/freezed_annotation.dart

```diff
@@ -11,6 +11,7 @@ class Freezed {
     this.unionKey,
     this.unionValueCase = FreezedUnionCase.none,
     this.fallbackUnion,
+    this.shouldGenerateMaybeMap = false,
```

Comment:
Do we really want this as the default behavior? Wouldn't it cause breaking changes?

Reply:
Yes I've changed it :)